### ISSUE TITLE
Build is dependent on python2

### ISSUE THREAD

**Issue by owengriffin** [OpenMTC/OpenMTC#4]
The build step:

```
get_requirements_from_setup_file ()
{
    # Each setup file is assumed to hold ".py" suffix, this gets
    # removed here
    local module_name=${setup_file%.py}

    cd ${working_dir}
    python - << END_OF_PYTHON
from importlib import import_module
from re import sub
setup = import_module('${module_name}', '${module_name}')
print('\n'.join(map(lambda x: sub('[\s+]', '', x),
                   setup.SETUP_INSTALL_REQUIRES)))
END_OF_PYTHON
}
```

causes a failure if the development system neither has python2.7 or python 2 setuptools installed locally. 

Results in the `requirements.txt` of pip not being installed in the resulting image.

**Comment by ronald-steinke** [comment]
Hi @owengriffin,

thanks for reporting. I will add a test in the beginning that will test it and give some hints on dealing with it.

Best,
Ronald

**Comment by rst-fokus** [comment]
Hi @owengriffin,

this should be fixed now with 8e6145d1e9363b6ab51938bd4c90d7fa3c4777f1.

Best,
Ronald
